File: swift/dataArrayTest.swift

func test(){
	let a = ["apple", "banana"]
	for k in a {
		print(k)
	}
}

func test2(){
	let a:[String] = ["apple", "banana"]
	print(a)
}

func test3(){
	var a = [String]()
	a.append("apple")
	a.append("banana")
	print(a)
}

func test4(){
    var a = ["apple", "banana", "orange", "longan"]
    a.remove(at: 1)
    print(a)
}

func test5(){
	var a = ["apple", "banana"]
	let b = ["orange", "pear"]
	a += b
	print(a)
}

func test6(){
	let a = [String](repeating: "apple", count: 2)
	print(a, a.count, a.isEmpty)
    print(a)
}

File: scala/dataArrayTest.scala

object Test{
	def main(args:Array[String]):Unit = {
		test()
	}

	def test():Unit = {
		var a = Array("apple", "banana", "orange")
		for(k <- a){
			println(k)
		}
	}

	def test2():Unit = {
		var a = new Array[String](3)
		a(0) = "apple"
		a(1) = "banana"
		a(2) = "orange"
		for(k <- a){
			println(k)
		}
	}
}